### ISSUE TITLE
IndexError: string index out of range

### ISSUE THREAD

**Issue by to175** [spyder-ide/spyder#12138]
### What steps will reproduce the problem?

I have just began to type "brow" for "browser" ... and Spyder encountered an internal problem. I have Kite installed.

So Why do I have this error please ?

### Traceback
```python-traceback
  File "C:\Users\theo1\Anaconda3\lib\site-packages\spyder\plugins\completion\fallback\actor.py", line 155, in handle_msg
    msg['current_word'])
  File "C:\Users\theo1\Anaconda3\lib\site-packages\spyder\plugins\completion\fallback\actor.py", line 62, in tokenize
    valid = is_prefix_valid(text, offset, language)
  File "C:\Users\theo1\Anaconda3\lib\site-packages\spyder\plugins\completion\fallback\utils.py", line 122, in is_prefix_valid
    current_pos_text = text[offset - 1]
IndexError: string index out of range
```

## Versions

* Spyder version: 4.1.1 
* Python version: 3.7.3
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.3.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.2.1 (OK)
diff_match_patch >=20181111    :  20181111 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.6.1 (OK)
jedi =0.15.2                   :  0.15.2 (OK)
nbconvert >=4.0                :  5.5.0 (OK)
numpydoc >=0.6.0               :  0.9.1 (OK)
paramiko >=2.4.0               :  2.7.1 (OK)
parso =0.5.2                   :  0.5.2 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.6.3 (OK)
pygments >=2.0                 :  2.4.2 (OK)
pylint >=0.25                  :  2.3.1 (OK)
pyls >=0.31.9;<0.32.0          :  0.31.9 (OK)
qdarkstyle >=2.8               :  2.8 (OK)
qtawesome >=0.5.7              :  0.5.7 (OK)
qtconsole >=4.6.0              :  4.7.2 (OK)
qtpy >=1.5.0                   :  1.8.0 (OK)
rtree >=0.8.3                  :  0.9.3 (OK)
sphinx >=0.6.6                 :  2.1.2 (OK)
spyder_kernels >=1.9.0;<1.10.0 :  1.9.0 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  18.0.0 (OK)

# Optional:
cython >=0.21                  :  0.29.12 (OK)
matplotlib >=2.0.0             :  3.1.0 (OK)
numpy >=1.7                    :  1.16.4 (OK)
pandas >=0.13.1                :  0.24.2 (OK)
scipy >=0.17.0                 :  1.2.1 (OK)
sympy >=0.7.3                  :  1.4 (OK)
```

**Comment by ccordoba12** [comment]
Duplicate of #11862

**Comment by ccordoba12** [comment]
It'll be fixed in our next version (4.1.2), to be released shortly.
